Sample Applications/WPFGallery/Views/SettingsPage.xaml.cs

```diff
@@ -19,5 +18,68 @@ public SettingsPage(SettingsPageViewModel viewModel)
             DataContext = this;
             InitializeComponent();
         }
+
+        private void Services_Click(object sender, RoutedEventArgs e)
+        {
+            Process.Start(new ProcessStartInfo("https://go.microsoft.com/fwlink/?LinkId=822631") { UseShellExecute = true });
+        }
+
+        private void Privacy_Click(object sender, RoutedEventArgs e)
+        {
+            Process.Start(new ProcessStartInfo("https://go.microsoft.com/fwlink/?LinkId=521839") { UseShellExecute = true });
+        }
+
+        private void Open_Issues(object sender, RoutedEventArgs e)
+        {
+            Process.Start(new ProcessStartInfo("https://github.com/microsoft/WPF-Samples/issues/new") { UseShellExecute = true });
+        }
+
+        private void Open_ToolkitInformation(object sender, RoutedEventArgs e)
+        {
+            Process.Start(new ProcessStartInfo("https://www.nuget.org/packages/CommunityToolkit.Mvvm/") { UseShellExecute = true });
+        }
+
+        private void Open_DIInformation(object sender, RoutedEventArgs e)
+        {
+            Process.Start(new ProcessStartInfo("https://www.nuget.org/packages/Microsoft.Extensions.DependencyInjection/") { UseShellExecute = true });
+        }
+
+        private void Open_HostingInformation(object sender, RoutedEventArgs e)
+        {
+            Process.Start(new ProcessStartInfo("https://www.nuget.org/packages/Microsoft.Extensions.Hosting") { UseShellExecute = true });
+        }
+
+        private void Change_ThemeMode_SelectionChanged(object sender, SelectionChangedEventArgs e)
```

Comment:
A better name would be ThemeMode_SelectionChanged / ThemeModeChanged, etc.
Name is a bit redundant here.

Reply:
Agreed, done!